custom_connectors/oauth2/bim_360.rb

```diff
@@ -3324,22 +2289,27 @@
                        'filter[type]=items&filter[lastModifiedTimeRollup]-ge=' \
                        "#{last_modified_time}&" \
                        "page[limit]=#{limit}&page[skip]=#{skip}"
-                     get("/data/v1/projects/#{project_id}/folders/" \
-                       "#{folder_id}/contents", query_params)
+                     get("/data/v1/projects/#{input['project_id']}/folders/" \
```

Comment:
you can use `project_id` field instead of `input['project_id']`

Reply:
this is where we were seeing issues where the `project_id` is cached from another trigger and was not polling from the project specified in the input. we're using `input['project_uid']` to make sure we're polling from the project specified in the input instead of the cached one.